recipes/ale/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e # Abort on error
+
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -DBUILD_TESTS=OFF ..
+cmake --build . --target install
+$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
```

Comment:
Generally, it is best to use `pip` for installs. Try something like `pip install . --no-deps -vv`.

Reply:
:+1: added `pip` to the host as well - perhaps should be in the build since it is required for the installation post make build?

Reply:
It needs to be in host IIRC.